CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [0.6.4] - 2018-02-15
+### Added
+- support for Django 2.0
```

Comment:
please make it 0.7.0 with info that support to Django 1.8, 1.9 i 1.10 is dropped